src/voice-pe/resources/pcb-design-files.md

```diff
@@ -6,6 +6,6 @@ zendesk:
   labels: voice pe, downloads, resources
 ---
 
-The repository with the source files is in preparation.
 
 - [Design files PDF](/static/docs/voice/ha_voice_preview_edition_pcb_design_files.pdf)
+- [GitHub repository with the KiCad design files](https://github.com/NabuCasa/home-assistant-voice-pe)
```

Comment:
Is "KiCad" well-known for people looking for this, or does it need an explanation?

Reply:
good question. I would have thought this is a known name among people looking for Design Files, but I could be wrong.